.github/workflows/release.yml

```diff
@@ -40,7 +40,7 @@ jobs:
             os: macos-latest
             vflags: -d cross_compile_macos_arm64
           - build_type: release
-            cflags: -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -frename-registers -ftree-vectorize
+            cflags: -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -frename-registers -ftree-vectorize -funroll-loops -ffunction-sections -fdata-sections
```

Comment:
All those, make executables smaller, and potentially faster, but at the expense of valuable information, that is lost, when there is a crash.

I think, that V analyzer is not stable enough, to benefit from them, while missing the debugging info harms collecting actionable diagnostic information for the issues, which will make stabilizing it slower.